### PR TITLE
fix: Snackbar is appearing in the middle of the screen in eventDetailFragment

### LINKED ISSUE #962: Snackbar is appearing in the middle of the screen in eventDetailFragment
**Describe the bug**
Error fetching Social Links is appearing in the middle of the screen

**Screenshots**
whatsapp image 2019-01-27 at 2 12 57 am

**Smartphone Info:**
|               | |
|---------------|-|
|Device         |[Redmi Note 4]|
|Android Version|[7.0]  |

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
fix #962   Snackbar is appearing in the middle of the screen in eventDetailFragment 

removed the coordinate layout from fragment_social_links
added app:layout_dodgeInsetEdges="bottom" top frameLayout of fragment_event so whole thing shifts up when snackbar is shown
ezgif com-video-to-gif

fix #962

### CODE REVIEW

**Comment by kshitijsuri90** [comment]
Mention which issue it fixes in the description.

File: app/src/main/res/layout/fragment_social_links.xml [hunk 1]
```diff
@@ -2,12 +2,8 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_social_links.xml]: Change this framelayout to coordinator layout

Comment (pranavpandey1998official) [reply app/src/main/res/layout/fragment_social_links.xml]: that was the bug if I changed it back then snackbar will take it as the first coordinate-Layout parent  and appear as before

Comment (pranavpandey1998official) [reply app/src/main/res/layout/fragment_social_links.xml]: if you want cordirdinate layout in view hiarchy we could nest it inside frameLayout

**Pull request merged**